src/pages/Admin/TagConfig/components/TagConfigTable.tsx

```diff
@@ -59,15 +59,8 @@ function TagConfigCard({
   const { t } = useTranslation();
   const [showArchiveDialog, setShowArchiveDialog] = useState(false);
 
-  const handleCardClick = () => {
-    onView(config.id);
-  };
-
   return (
-    <Card
-      className="cursor-pointer hover:shadow-md transition-shadow"
-      onClick={handleCardClick}
-    >
+    <Card className="hover:shadow-md transition-shadow">
```

Comment:
```suggestion
    <Card>
```

Card is only for mobile, I do no think these styles work `hover:shadow-md transition-shadow` on mobiles

Reply:
@copilot what do you think ? 👆